cpg_workflows/workflow.py

```diff
@@ -668,7 +702,17 @@ def _get_action(self, target: TargetT) -> Action:
         logging.info(f'{self.name}: {target} [QUEUE]')
         return Action.QUEUE
 
-    def _is_reusable(self, expected_out: ExpectedResultT) -> tuple[bool, Path | None]:
+    def _is_reusable(
+        self, expected_out: ExpectedResultT, existing_outputs: set[Path] | None = None
+    ) -> tuple[bool, Path | None]:
+        """
+        Checks if the outputs of prior stages already exists, and can be reused
```

Comment:
```suggestion
        Checks if the outputs of prior stages already exist, and can be reused
```

Also what is the policy on filling in `Args` and `Returns` in these sorts of comments…?

Reply:
Ah I definitely left that one dry. I always try and populate with type/description where possible. It's a shame that neither Pycharm or Copilot seem capable of taking the type hints from the method definition and including them in the docstring.